ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master__packetcheck.yaml

```diff
@@ -0,0 +1,31 @@
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: images/Dockerfile.ci
```

Comment:
No need to rebuild the `dev-scripts` image

Reply:
Same as above.